custom_components/tuya_local/devices/morray_level_sensor.yaml

```diff
@@ -0,0 +1,109 @@
+name: Level sensor
+products:
+  - id: ywcgq
+    manufacturer: MORRAY
```

Comment:
The brand seems to be "Moray", with one r and not an abbreviation so no need for all-caps.